client/js/shout.js

```diff
@@ -382,6 +386,20 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("highlights") !== null) {
```

Comment:
Yes, I think this would be the best way to do things. I've done this in #603, let me know what you think about how it's done there. I can just do it the same way.